src/commands/anime/anime.js

```diff
@@ -51,31 +54,120 @@ function getAnimeInfos(anime) {
 	infosFields.push({
 		name: 'Rating :star:',
 		value: `${anime.attributes.averageRating}/100`,
+		inline: true,
+	});
+
+	infosFields.push({
+		name: 'Content Rating :underage:',
+		value: `${anime.attributes.ageRatingGuide}`,
+		inline: true,
 	});
 
 	return infosFields;
 }
 
+function sendAnimeInfos(msg,anime) {
+	msg.channel.send({
+		embed: {
+			title: `:flag_gb: ${anime.attributes.titles.en_jp}\n:flag_jp: ${anime.attributes.titles.ja_jp}`,
+			image: {
+				url: `${anime.attributes.coverImage != null ? anime.attributes.coverImage.original : ''}`,
+			},
+			thumbnail: {
+				url: `${anime.attributes.posterImage != null ? anime.attributes.posterImage.original : ''}`,
+			},
+			fields: getAnimeInfos(anime),
+			description: `**Synopsis : **\n${anime.attributes.synopsis.split('[')[0].split('(Source')[0]}\n`,
+		},
+	});
+	
+}
+
+function getAnimesListInfos(animesList){
+	const animesFields = [];
+	for(var i = 0; i < animesList.length; i++){
```

Comment:
Please, use a `let` variable over here. I don't use any `var` variable in this project. :)